apps/frontend/app/routes/events+/$id+/_index.tsx

```diff
@@ -208,88 +214,72 @@ const PageComponent: FC = () => {
 
   const [selection, setSelection] = useState(choice.dish)
 
-  const menu = useMemo(
-    () => menus.find((menu) => menu.id === selection),
-    [menus, selection],
-  )
-
   useEffect(() => {
     setSelection(choice.dish)
   }, [choice.dish])
 
   return (
     <>
-      <header className="mb-4">
+      <header className="mb-4 w-full flex flex-row justify-center items-center">
         <h1 className="text-2xl">{title}</h1>
         <h2>{date}</h2>
       </header>
-      <div className="grid grid-cols-3 gap-x-5">
-        <div className="grid col-span-2 grid-cols-2 gap-5">
+      <div className="px-3 w-full flex flex-row justify-center">
+        <div className="grid gap-2 sm:grid-cols-1 lg:grid-cols-2 max-w-2xl m-1">
           {menus.map((menu) => {
             const selected = menu.id === selection
-
-            const highlight =
-              selected && menu.id === choice.dish ? true : selected
-
             return (
-              <article key={menu.id} className="flex flex-col gap-y-4">
-                <header className="flex justify-between">
-                  <div>
-                    <h1 className="text-xl">{menu.name}</h1>
-                    <p>{menu.description}</p>
-                  </div>
-                  <div>
-                    {highlight === false && (
-                      <Form method="post">
-                        <input type="hidden" name="dish" value={selection} />
-                        <Submit>Choose</Submit>
-                      </Form>
-                    )}
+              <div>
```

Comment:
```suggestion
              <div key={menu.id}>
```

Attention quand tu fais un ".map" pour render un element, il faut pas oublié l'attribut "key" pour eviter les rerender.

Reply:
done